src/ShopCurrencyBundle/Bridge/Chameleon/Objects/CurrencyBasket.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace ChameleonSystem\ShopCurrencyBundle\Bridge\Chameleon\Objects;
+
+use ChameleonSystem\CoreBundle\ServiceLocator;
+use ChameleonSystem\ShopCurrencyBundle\Interfaces\ShopCurrencyServiceInterface;
+
+class CurrencyBasket extends \ChameleonSystemShopCurrencyBundleBridgeChameleonObjectsCurrencyBasketAutoParent
+{
+    /**
+     * Reload active set payment method on currency change.
```

Comment:
How about "Reloads the active payment method on currency change, so that the payment method holds the payment charges in the correct currency."?

Reply:
ok